source/codegen/templates/service_helpers.mako

```diff
@@ -690,15 +690,17 @@ ${initialize_standard_input_param(function_name, parameter)}
 
 
 ## Handles populating the response message after calling the driver API.
-<%def name="populate_response(output_parameters, indent_level=0, init_method=False)">\
+<%def name="populate_response(function_data, output_parameters, indent_level=0, init_method=False)">\
 <%
   get_last_error_outputs = service_helpers.get_last_error_output_params(output_parameters)
   normal_outputs = [p for p in output_parameters if not p in get_last_error_outputs]
+  return_type = service_helpers.get_function_return_type(function_data)
```

Comment:
Instead of making this change here, you could add `'status_expression' : '0'` attribute on `StatusToString` API. That way in metadata we are tagging the function to always succeed and status variable will be generated (with value of `status_expression`) for this API as well. Then you don't need this additional check of setting `response->status` differently here.

Reply:
yeah you are right. I was not aware of this. I have made the change.